fastapi_crudrouter/core/sqlalchemy.py

```diff
@@ -86,11 +86,11 @@ def route(db: Session = Depends(self.db_func)):
         return route
 
     def _delete_one(self) -> Callable:
-        def route(item_id: int, db: Session = Depends(self.db_func)):
+        def route(item_id, db: Session = Depends(self.db_func)):
```

Comment:
What if we did something like below?

```suggestion
        def route(item_id: Union[int, str], db: Session = Depends(self.db_func)):
```

Reply:
This would allow for both int and string primary keys